src/renderer/screens/exchange/Swap2/Form/FormSelectors/FormInputs.js

```diff
@@ -27,7 +27,7 @@ export default function FormInputs() {
   const [toAmount, setToAmount] = useState(null);
 
   return (
-    <>
+    <section>
```

Comment:
What's the benefit of wrapping this in a section as opposed to leaving it flat? 🤔

Reply:
In my opinion, as it's an encapsulated component built to be injected elsewhere, returning only one node-point instead of a list of nodes creates a better HTML structure and improve composability. For example, in this case, the parent element is free to define gap rules (as I do here by setting a row gap rule) to manage the layout without creating side effects to the injected child component. If I'm not putting the section tag here to encapsulate the structure of this component, I should then have added it from all parent components that use this child one depending on the parent CSS rules.

From me, that's the same reason why we try to avoid setting margin rules to the container of a component, that improves composability.

Reply:
I'm ok with it, just wanted your reasoning :)